README.md

```diff
@@ -30,78 +29,89 @@ version: 2.1
 
 orbs:
```

Comment:
Open to conversation. I prefer to point to the orb registry for the included usage examples so that we do not need to manage them in two locations.